frontend/src/features/verifiable-fields/Email/VerifiableEmailField.tsx

```diff
@@ -0,0 +1,61 @@
+import { baseEmailValidationFn } from '~utils/fieldValidation'
+import {
+  EmailFieldInput,
+  EmailFieldProps,
+  EmailFieldSchema,
+} from '~templates/Field/Email'
+
+import { VerifiableFieldContainer } from '../components/VerifiableFieldContainer'
+import { VerifiableFieldSchema } from '../types'
+import { useVerifiableField } from '../VerifiableFieldContext'
+import { VerifiableFieldProvider } from '../VerifiableFieldProvider'
+
+export type VerifiableEmailFieldSchema = VerifiableFieldSchema<EmailFieldSchema>
+
+export interface VerifiableEmailFieldProps extends EmailFieldProps {
+  schema: VerifiableEmailFieldSchema
+}
+
+/**
+ * @example
+ * { value: '+65 9876 5432', signature: some-signature }
+ */
+const InnerVerifiableEmailField = ({
+  schema,
+  ...formContainerProps
+}: VerifiableEmailFieldProps): JSX.Element => {
+  const { handleInputChange, handleVfnButtonClick } = useVerifiableField()
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Enter') {
```

Comment:
Does this need to additionally account for tab (blur event)?

Reply:
Nope, tab shouldn't trigger the opening of the verification box, only Enter